src/ucs/Makefile.am

```diff
@@ -5,7 +5,7 @@
 # See file LICENSE for terms.
 #
 
-SUBDIRS = vfs/sock . vfs/fuse
+SUBDIRS = vfs/sock . vfs/fuse . signal
```

Comment:
why added '.' another time?

Reply:
> why added '.' another time?

to be sure libucs_signal is built after libucs...
removed